packages/components/fields/multiline-text-field/src/multiline-text-field.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import type { Meta, StoryFn } from '@storybook/react';
+import { iconArgType } from '@/storybook-helpers';
+import MultilineTextField from './multiline-text-field';
+import { useState } from 'react';
+
+const meta: Meta<typeof MultilineTextField> = {
+  title: 'Form/Fields/MultilineTextField',
+  // @ts-expect-error, fix component and/or types
+  component: MultilineTextField,
+  argTypes: {
+    // @ts-expect-error
+    title: { control: 'text' },
+    hint: { control: 'text' },
+    description: { control: 'text' },
+    hintIcon: iconArgType,
+  },
+};
+export default meta;
+
+type Story = StoryFn<typeof MultilineTextField>;
+
+export const BasicExample: Story = (args) => {
+  // @ts-expect-error
+  const { defaultExpandMultilineText } = args;
+  const [value, onChange] = useState('');
```

Comment:
I understand why, but I should point out that the `value` control doesn't do anything on this story
![Screenshot 2024-08-15 at 4 18 21 PM](https://github.com/user-attachments/assets/fee047af-e411-45a6-8e96-f4108889f34b)

Reply:
Good catch. [I made it work](https://github.com/commercetools/ui-kit/pull/2880/commits/0cea1d8300db73a6641bedcb1f1fbe7a176a047d).

Reply:
Almost there! 👏
@misama-ct thanks for the fix! I was doing another quick round of review and I found that if `isMulti` is true and the `value` object is clicked, an empty object is rendered in the Input. 
![2024-08-16 10 21 41](https://github.com/user-attachments/assets/18298539-50cf-4db0-92bc-e6582c5d9c4e)

Reply:
@ddouglasz What exactly would be your expectation? Checking the component, to me it behaves as expected, as the initial payload is simply not in the shape that the field expects. As soon as you define a json structure there (Array with object containing a label + value prop) it displays properly.

Reply:
Yes, it works as expected @misama-ct . It's just the initialisation is a bit weird on the UI before a value is set. 
Maybe this is a stretch, but I would expect that nothing happens on the interface until the key-value pair have been set so we can prevent the weird empty tag when no value has been set yet.

Reply:
We'd probably have to touch storybook internals for that, something we do want to avoid. Only thing I can offer for now is to deactivate the `value` prop, which may even make sense, cause even if you manage to configure the value-prop. The Component is then in some sort of _"controlled"_ mode, and the initially given functionality of selecting an item from the suggestion-list after entering some character is not working anymore. What do you think, is deactivating the `value` control an option?

Reply:
I think deactivating the value prop from the control should be fine. Since the user can already set values by interacting with the UI, see the json output and also code example. In my opinion, all these are already sufficient to provide the user with enough context for usage example.

Reply:
I [agree](https://github.com/commercetools/ui-kit/pull/2880/commits/2bf86e4233033e511441127c02718bd60d9d2c71).